card/mod/standard/set/all/rich_html/menu.rb

```diff
@@ -22,7 +22,8 @@
   end
 
   def menu_path_opts
-    opts = { slot: { home_view: voo.home_view } }
+    opts = { slot: { home_view: (voo.home_view || @slot_view),
```

Comment:
I always forget what home view means. Is it the view used for the main card? Maybe we want always @slot_view here.